src/Views/Homepage.vala

```diff
@@ -332,7 +332,9 @@ public class AppCenter.Homepage : Adw.NavigationPage {
             var installed = false;
             foreach (var origin_package in package.origin_packages) {
                 try {
-                    if (yield AppCenterCore.FlatpakBackend.get_default ().is_package_installed (origin_package)) {
+                    warning ("CHECK INSTALLED");
+                    if (AppCenterCore.FlatpakBackend.get_default ().is_package_installed (origin_package)) {
+                        warning ("CHECKED INSTALLED");
```

Comment:
Probably should remove these or at least set to debug if it's necessary :sweat_smile:

Reply:
Oops yeah they were just for testing :)